consolelabs/mochi-api
# chore: add swagger docs

**What does this PR do?**

-   [x] Add swagger docs for apis

**How to test**

-  run `make gen-swagger`
-   run `make dev`
- Go to http://localhost:8200/swagger/index.html#/ to check the swagger docs
